### DISCUSSION TITLE
QC: Rename `Transform_Rate()` and `Transform_Count()`

### DISCUSSION THREAD

**Discussion by Zhongkai-Wang** [Gilead-Public/gsm#1157] [Ideas]
## QC Details
`Transform_Rate()` is used for binary and rate outcomes. Here, "Rate" in "Transform_Rate" only means division with numerator and denominator. It's causing confusion multiple times.

`Transform_Count()` is only used for identity method.

## Suggestion
Rename `Transform_Rate()` as `Transform_Divide()` or  `Transform_Division()`. Glad to hear other proposals.

**Comment by mattroumaya** [comment]
@jwildfire @gwu05 @Zhongkai-Wang @samussiah - converted this to a discussion. As new folks are introduced to `{gsm}` we can consider this change/solicit feedback.
